nova_vm/src/ecmascript/types/language/string.rs

```diff
@@ -310,6 +310,44 @@ impl String {
         }
     }
 
+    pub fn get_property_descriptor(
```

Comment:
**nitpick:** I'd keep this `pub(crate)` as it's not really a nice public API.

Reply:
Done.